tools/benchmark/cmd/put.go

```diff
@@ -150,3 +160,42 @@ func max(n1, n2 int64) int64 {
 	}
 	return n2
 }
+
+func hashKV(cmd *cobra.Command, clients []*v3.Client) {
+	var host string
+	var epHashKVRev int64
+	eps, err := cmd.Flags().GetStringSlice("endpoints")
+	if err != nil {
+		panic(err)
+	}
+	for i, ip := range eps {
+		eps[i] = strings.TrimSpace(ip)
+	}
+	st := time.Now()
+	var results string
+	ctx, cancel := context.WithCancel(context.Background())
+	clients[0].HashKV(ctx, eps[0], epHashKVRev)
+	if !strings.HasPrefix(eps[0], `http://`) {
+		host = "http://" + eps[0]
```

Comment:
Why do we need this?

Reply:
@gyuho hi, I thought we check it by convention. It's not needed as you pointed out. I will remove it. Thanks!